torchci/components/TorchAgentPage.tsx

```diff
@@ -69,6 +85,13 @@ export const TorchAgentPage = () => {
   const [error, setError] = useState("");
   const [debugVisible, setDebugVisible] = useState(false);
 
+  // Chat history state
+  const [chatHistory, setChatHistory] = useState<ChatSession[]>([]);
+  const [selectedSession, setSelectedSession] = useState<string | null>(null);
+  const [isHistoryLoading, setIsHistoryLoading] = useState(false);
+  const [isSessionLoading, setIsSessionLoading] = useState(false);
+  const [drawerOpen, setDrawerOpen] = useState(true);
```

Comment:
setDrawerOpen is not used?

Reply:
will remove / start using as part of https://github.com/pytorch/test-infra/pull/6785